cardano-cli/test/Test/Cli/JSON.hs

```diff
@@ -54,10 +55,9 @@ genWord64 :: Gen Word64
 genWord64 = Gen.word64 Range.constantBounded
 
 genUTCTime :: Gen UTCTime
-genUTCTime =
- systemToUTCTime <$>
-   (MkSystemTime <$> Gen.int64 Range.constantBounded
-                 <*> Gen.word32 Range.constantBounded)
```

Comment:
Indeed, the old code was generating nanoseconds within a second (i.e. 0..1e9) using a range of 0..2^32-1, which makes no sense.

I fail to see what this has to do with leap seconds. Neither the SystemTime nor UTCTime functions know anything about leap seconds.

Reply:
The range `(1E9; 2E9)`  is used to represent a leap second. The values above `2E9` would indicate more than one second, which is not in the UTC standard. I've provided more details in the description.

Reply:
Oh ok, that makes sense. Thanks